_posts/en/newsletters/2022-08-31-newsletter.md

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Bitcoin Optech Newsletter #215'
+permalink: /en/newsletters/2022/08/31/
+name: 2022-08-31-newsletter
+slug: 2022-08-31-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal for a standardized wallet
+label export format and includes our regular sections with summaries of
+recent questions and answers from the Bitcoin StackExchange, a list of
+new software releases and release candidates, and descriptions of
+notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Wallet label export format:** Craig Raw [posted][raw
+  interchange] to the Bitcoin-Dev mailing list a proposed BIP to
+  standarize the format wallets use to export labels for their addresses
+  and transactions.  A standardized export format could in theory allow
+  two pieces of wallet software that used the same [BIP32][topic bip32]
+  account hierarchy to open each others' backups and recover not just
+  funds but all of the information the user manually entered about their
+  transactions.
+
+    Given previous challenges to making BIP32 wallets compatible with
+    each other, a perhaps more achievable use would be making it easier
+    to export transaction history from a wallet and use it in other
+    programs, such as for accounting.
+
+    Developers Clark Moody and Pavol Rusnak [each][moody slip15] replied
+    with a [reference][rusnak slip15] to [SLIP15][], which describes the
+    open export format developed for Trezor brand wallets.  Craig Raw
+    [noted][raw slip15] several significant differences between what his
+    proposed format is attempting to achieve and what SLIP15 seems to
+    provide.  Several other design aspects were also discussed, with
+    discussion appearing to be ongoing as this summary was being
+    written.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Why isn't it possible to add an OP_RETURN (or some arbitrary script) inside a taproot script path with a descriptor?]({{bse}}114948)
+  Antoine Poinsot explains that [script descriptors][topic descriptors] are
+  currently being extended to use [miniscript][topic miniscript] in Bitcoin
+  Core and expected in the Bitcoin Core 24.0 release. While initial miniscript
+  features will only include segwit v0 support,
+  eventually support for [tapscript][topic tapscript] could be added and
+  discussions about [partial descriptors][Bitcoin Core #24114] are currently underway.
```

Comment:
TBH, this sounds like an interesting topic, but I have read the answer multiple times now, and I'm still not sure I understand what it's saying about OP_RETURN in scriptpaths. I'm gonna see if I can try to clarify it tomorrow in the office.

Reply:
Ill leave as is for the newsletter for today. Let me know if there is feedback from your discussions.

Reply:
Yeah, it's fine in the newsletter, I just found the part about why it doesn't make a lot of sense to write an `OP_RETURN` into the script tree easy to follow. I wrote my own answer today: https://bitcoin.stackexchange.com/a/115100/5406